internal/controller/finbackupconfig_controller.go

```diff
@@ -2,34 +2,52 @@ package controller
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+	"time"
 
+	finv1 "github.com/cybozu-go/fin/api/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/log"
-
-	finv1 "github.com/cybozu-go/fin/api/v1"
 )
 
+var errEmptyClusterID error = errors.New("cluster ID is empty")
+
 // FinBackupConfigReconciler reconciles a FinBackupConfig object
 type FinBackupConfigReconciler struct {
 	client.Client
-	Scheme *runtime.Scheme
+	Scheme               *runtime.Scheme
+	finCephClusterID     string
```

Comment:
Let's make this name clearer. For example:


```suggestion
	managedCephClusterID     string
```